nsis/src/nsis/file/NsisConstants.java

```diff
@@ -6,10 +6,14 @@
 			(byte) 0xde };
 	public static final byte[] NSIS_MAGIC = { 'N', 'u', 'l', 'l', 's', 'o', 'f', 't', 'I', 'n', 's',
 			't' };
-	public static final int NB_NSIS_BLOCKS = 8;
 	public static final int NSIS_MAX_STRLEN = 1024;
 	public static final int NSIS_MAX_INST_TYPES = 32;
 	public static final byte COMPRESSION_LZMA = 0x5d;
 	public static final byte COMPRESSION_BZIP2 = 0x31;
 	public static final int COMPRESSION_LZMA_HEADER_LENGTH = 5;
+
+	public enum Blocks {
+		NB_PAGES, NB_SECTIONS, NB_ENTRIES, NB_STRINGS, NB_LANGTABLES, NB_CTLCOLORS, NB_BGFONT,
```

Comment:
Please spell out BGFONT and CTLCOLORS

Reply:
I have no clue what they mean

https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Source/exehead/fileform.h#l276

Do you have any idea?

Reply:
I think BGFONT would be background font but I'm not sure

Reply:
Yes, I think BGFONT is background font and CtlColors is "Control colors" (from this little snippet: https://github.com/kichik/nsis/blob/5e8634a42871785a08fe7425395039e1d1a0e18c/Contrib/nsDialogs/welcome.nsi#L109)

Reply:
Well spotted